.github/workflows/notebook_integration_tests.yaml

```diff
@@ -0,0 +1,46 @@
+name: SDK integration tests
+
+on:
+  push:
+    branches:
+      - main
+      - "fix-**"
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch:
+
+jobs:
+  integration:
+    name: Notebook integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+
+      - name: Create a virtual environment
+        run: uv venv
+        working-directory: notebooks
```

Comment:
we might need to specify the full working-dir path here: https://github.com/mozilla-ai/lumigator/blob/b2d760b13b6b8569c0a80e1252528e0b62edc7a6/.github/workflows/sdk_integration_tests.yaml#L32C28-L32C50

Reply:
@veekaybee I don't follow. I'm creating the venv in `/notebooks` and installing the `requirements.txt` that exists in this directory. What is the full working-dir path you're referring to?

Reply:
Does this step (and others) need to know about the fully-qualified path here, aka `lumigator/notebooks`?

Reply:
I don't think so. I have built the venv in /`notebooks` that's why I'm invoking every other step from this directory. For example:

```
      - name: Install sdk
        run: uv pip install -e ../lumigator/python/mzai/sdk
        working-directory: notebooks
```